extensions/resteasy-reactive/rest-client-reactive-jackson/runtime/src/main/resources/quarkus-extension.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: "MicroProfile Rest Client with RESTEasy Reactive and Jackson"
```

Comment:
The equivalent `rest-client-jackson` has `REST Client Jackson` for name, whereas this one seems to tell me this extension is a full rest client, but it's not, it's support for jackson for the rest client:

```
name: "REST Client Jackson"
metadata:
  keywords:
  - "rest-client-jackson"
  - "rest-client"
  - "web-client"
  - "microprofile-rest-client"
  - "json"
  - "jackson"
  categories:
  - "web"
  - "serialization"
  status: "stable"
```

Reply:
I'm not sure I follow. Both extensions, quarkus-rest-client-jackson and quarkus-rest-client-reactive-jackson bring in the client extension. It's enough to add the `-jackson` extension to the project to use the client. 
So for the content of the extension point of view, indeed it just adds jackson to the client. But for users, adding the extension adds the client with jackson support to their project.

OTOH, the current name is rather long :)

Reply:
I guess I agree with you, but this is not how it's described for RESTEasy Classic. We have to harmonise both.

Reply:
I reworked it to align names and descriptions to the way RestEasy (classic) server extensions do it.